tensorboard/plugins/profile/overview_page/BUILD

```diff
@@ -0,0 +1,17 @@
+package(default_visibility = ["//tensorboard:internal"])
+
+load("//tensorboard/defs:web.bzl", "tf_web_library")
+
+licenses(["notice"])  # Apache 2.0
+
+tf_web_library(
+    name = "overview_page",
```

Comment:
BTW, thank you for organizing the tools into separate components.

Reply:
No problem.